nano/node/network.cpp

```diff
@@ -891,32 +891,35 @@ void nano::tcp_message_manager::put_message (nano::tcp_message_item const & item
 {
 	{
 		nano::unique_lock<std::mutex> lock (mutex);
-		while (entries.size () > max_entries && !stopped)
+		while (entries.size () >= max_entries && !stopped)
```

Comment:
max_entries seems like it should be a valid size, did we need to change this comparison?

Reply:
Without the `=` then it's possible to go 1 above `max_entries`. Arguably it could then just be equality `while (entries.size () == max_entries)` because this is the only function which appends entries, so it would never go above the maximum (but possible fail-safe for future changes I guess?).

Reply:
Initially the added unit test had to target `max_entries + 1` rather than `max_entries`. Not a significant change but it's technically correct ;)